source/index.html

```diff
@@ -97,7 +90,7 @@ <h2 class="feachures__title">Как это работает</h2>
           </p>
         </li>
         <li class="feachures__item feachures__item--time">
-          <p> Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+          <p class="feachures__description"> Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
           </p>
```

Comment:
Надо или закрыть на строке
```suggestion
          <p class="feachures__description">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
```

Ну или разнести уж
```suggestion
          <p class="feachures__description">
            Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
          </p>
```